openedx/core/djangoapps/config_model_utils/models.py

```diff
@@ -167,13 +227,13 @@ def current(cls, site=None, org=None, org_course=None, course_key=None):  # pyli
             raise ValueError("Only one of site, org, org_course, and course can be specified")
 
         if org_course is None and course_key is not None:
-            org_course = cls._org_course_from_course_key(course_key)
+            org, org_course = _org_course_from_context_key(course_key)
```

Comment:
I don't think a non-None value will ever be overwritten since:
- We check earlier in the method that only one of `site`, `org`, `org_course`, and `course_key` are provided. 
- So if `course_key` is not `None`, then `org` definitely is.

Reply:
Alright, I think I see it now; I guess the (pre-existing) logic is over-complicated (eg, above, that existing check could be `if  course_key is not None:`, since only one value can be non-None).

Nothing to see here!